virttest/qemu_vm.py

```diff
@@ -1694,6 +1709,13 @@ def sort_key(dev):
                                     numa_mem, numa_cpus, numa_nodeid)
             devices.insert(StrDev('numa', cmdline=cmdline))
 
+        for numa_node in params.objects("guest_numa_nodes"):
+            numa_params = params.object_params(numa_node)
+            numa_nodeid = numa_params.get("numa_nodeid")
+            numa_dist = ast.literal_eval(numa_params.get("numa_dist", "[]"))
+            if numa_nodeid is not None and numa_dist:
+                add_numa_dist(devices, numa_nodeid, numa_dist)
```

Comment:
Is it necessary to pass devices here?

Reply:
It is not must, without `devices` it still can work,  just keep align with other functions that used to add device.

Now, i drop this function and move it out, could you help re-review, thanks!